### DISCUSSION TITLE
同一个资源，win10上正常还可快，但在安卓上用就不行。

### DISCUSSION THREAD

**Discussion by dylanleesky** [2dust/v2rayNG#3477] [General]
同一个资源，同一个网络，
win10用，正常，速度还挺快。
安卓用，测试1000+毫秒，但什么特殊页都打不开，微信消息出现滞后、策略是绕过局域网和大陆。

但是，若是在无线中设置手动代理，用win10做代理服务器去访问google啥的，速度也挺快。

请教这是怎么回事，该怎么办。

**Comment by 2dust** [comment]
是否能用的问题移到讨论去
